drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/azure/azure-storage-class.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: elasticsearch-sc
+provisioner: kubernetes.io/azure-disk
```

Comment:
Use the ` disk.csi.azure.com` provisioner here? 

Link:  https://learn.microsoft.com/en-us/azure/aks/azure-disk-csi

Reply:
✅